src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -451,8 +451,17 @@
             batchFieldOptions.fieldGroups.push(currentFieldGroup);
 
         });
+
+        let selectedMatchingRule = component.get('v.batchInfo.donationMatchingRule');
+        let donationMatchingOptionValues = donationMatchingOptions.map(function(option) { return option.value });
+        // Filter out any selected matching rules that aren't selected as available matching fields
+        selectedMatchingRule = selectedMatchingRule.filter(function (selectedRule) {
+            return (donationMatchingOptionValues.indexOf(selectedRule) >= 0);
+        });
+
```

Comment:
@ptewson-sfdo @kenslewis @bethbrains - this is the commit I mentioned during stand up today.

Reply:
This approach looks good - nice work turning that around so quickly